bin/node/src/chain_spec.rs

```diff
@@ -131,6 +131,10 @@ pub struct ChainParams {
     #[arg(long, value_parser = parse_account_id, default_value(DEFAULT_SUDO_ACCOUNT))]
     sudo_account_id: AccountId,
 
+    /// AccountIds of the optional rich accounts
+    #[arg(long, value_delimiter = ',', value_parser = parse_account_id, num_args=1..)]
+    rich_account_ids: Option<Vec<AccountId>>,
```

Comment:
why not just `Vec<AccountId>`? and empty vector is pretty much the same as `None`

Reply:
using Option for an optional arguments makes the intent clearer